src/backend/projects/project_crud.py

```diff
@@ -19,19 +19,37 @@
 
 import geojson
 import io
-import json
+import os
+from json import loads, dumps, dump
 from typing import List
 from zipfile import ZipFile
+import epdb
 from fastapi import HTTPException, UploadFile
 from geoalchemy2.shape import to_shape
-from geojson_pydantic import FeatureCollection
-from shapely.geometry import mapping, shape
+import geoalchemy2
+import numpy as np
```

Comment:
Well dang, there it is. I always try to avoid bringing in Numpy, Pandas, and friends as long as possible, but I guess we've hit that point where they inevitably show up!

Reply:
Pandas we're not using geoalchemy and  numpy were already being used, just not by this file yet.

Reply:
Ah, got it! Glad to hear we've not yet been invaded by black and white bears! (I'm sure they'll show up eventually)

Reply:
I uses xmltodict instead of pandas.